test/unit/shared/access_permissions_test_base.rb

```diff
@@ -5,15 +5,19 @@ module AccessPermissionsTestBase
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def check_routes(app_routes, skipped_actions)
+    def should_skip_path?(path, skipped_actions = [], skip_patterns = [])
+      skipped_actions.include?(path) || skip_patterns.any? { |pattern| path =~ pattern }
+    end
+
+    def check_routes(app_routes, skipped_actions, skip_patterns: nil)
```

Comment:
Should this be 
```suggestion
    def check_routes(app_routes, skipped_actions, skip_patterns: [/^(api\/v2\/)?(dummy|fake)_/])
```

Reply:
Yeah, probably... I wanted to allow passing `nil`, but there is no reason for it.